src/jagrati/Assigment23.java

```diff
@@ -0,0 +1,28 @@
+package jagrati;
+
+public class Assigment23 {
+	
+	void getCountChar(String str) {
+			int uCount =0; 
+			int lCount =0;
+			int dCount =0;
+			int sCount =0;
+			for(int index=0; index < str.length(); index++) {
+				if (Character.isUpperCase(str.charAt(index))) {
```

Comment:
It would be better if you create char ch = str.charAt(index) outside of condition, so you no need to write it again and again, you have to just write ch

Reply:
Sure Margi .... will keep in mind in next assignment thank you for review comment